.vscode/settings.json

```diff
@@ -25,7 +25,6 @@
     },
     "files.eol": "\n",
     "todo-tree.filtering.excludeGlobs": ["**/node_modules/**", "**/dist/**", "**/dist-test/**", "**/lib-web/**", "**/*.min.*", "**/*.map"],
-    "todo-tree.general.tags": ["TODO:"],
```

Comment:
please re-add

Reply:
this would prevent comments like 

// TODO something

to be recognized becaus of the missing ":" was this intenional ?

Reply:
yep, the setup of all our projects looks the same.